pom.xml

```diff
@@ -87,4 +87,31 @@
       </plugin>
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>openshift-it</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <configuration>
+              <systemPropertyVariables>
+                <app.name>${project.artifactId}</app.name>
```

Comment:
Why does `app.name` have to be set both in pom.xml and arquillian.xml?

Reply:
Technically it doesn't, only setting it in `systemPropertyVariables` would do the trick.

I'll go ahead and update the PR

Reply:
Fixed

Reply:
And the opposite is not the case? Having test properties in arquillina.xml would make a nice separation. But whatever works is fine I suppose.

Reply:
Initially I was hoping to have it only in arquillian.xml, but that didn't work out, so having it there was an incorrect leftover

Reply:
Exactly. I initially wanted to have it only in arquillian.xml, but I couldn't then inject it's value into the test. 
So I had to go with the pom solution :(